tasks/test.yml

```diff
@@ -41,15 +46,24 @@ tasks:
   phpstan:
     dec: Runs PHPStan with mglaman/phpstan-drupal
     cmds:
-      # @todo allow the config to be overridden locally.
-      - ./vendor/bin/phpstan analyse -c vendor/lullabot/drainpipe-dev/config/phpstan.neon {{.PHPSTAN_DIRS}}
+      - ./vendor/bin/phpstan analyse -c vendor/lullabot/drainpipe-dev/config/phpstan.neon {{.TEST_DIRS}}
   phpunit:
     desc: Runs PHPUnit
     cmds:
-      # Ensure directories exist so PHPUnit doesn't complain.
       - |
-        DIRS_ARR=({{.PHPUNIT_DIRS}})
-        for DIR in "${DIRS_ARR[@]}"; do
-          mkdir -p $DIR
-        done
-      - ./vendor/bin/phpunit {{.PHPUNIT_DIRS}}
+        {{ .FUNC_ENSURE_DIRS }}
+      - ./vendor/bin/phpunit {{.TEST_DIRS}}
+  phpcs:
+    desc: Runs PHPCS with Drupal Coding Standards
+    cmds:
+      - |
+        {{ .FUNC_ENSURE_DIRS }}
+      - ./vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer
```

Comment:
If I remember right, this is a global setting that could affect all environments on a local. Would it be better for us to create `phpcs.xml.dist` in the project root, that then includes the coder sniffer standards?